web/src/features/App/api/useLicense.ts

```diff
@@ -7,24 +7,20 @@ function useLicenseWithIntercept() {
 
   let timerId = useRef<null | NodeJS.Timeout>(null);
```

Comment:
timeout is stored as a `ref` rather than a `state` variable so that async functions can grab the latest version. https://dmitripavlutin.com/react-hooks-stale-closures/